t/plugin/proxy-rewrite3.t

```diff
@@ -455,3 +455,12 @@ passed
 GET /echo HTTP/1.1
 --- response_headers
 test: test_in_set
+
+
+
+=== TEST 21: deprecated calls test
```

Comment:
We can merge this case to `TEST 20`?

Reply:
I think each test should foucus on one function point, even if their trigger conditions are same.